gluefactory/models/backbones/dinov2.py

```diff
@@ -10,6 +10,7 @@ class DinoV2(BaseModel):
 
     def _init(self, conf):
         self.net = torch.hub.load("facebookresearch/dinov2", conf.weights)
+        self.set_initialized(True)
```

Comment:
```suggestion
        self.set_initialized()
```
and replace other similar calls